src/javascript/app/pages/trade/purchase/contract-details.jsx

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable import/no-unresolved */
+import React, { useEffect, useState } from 'react';
+import { Button,  Skeleton, Text } from '@deriv-com/quill-ui';
+import { LabelPairedArrowLeftMdRegularIcon } from '@deriv/quill-icons/LabelPaired';
+import ContractTable from './contract-table';
+import Portal from '../../portal';
+import purchaseManager from '../../../common/purchase_manager';
+import { usePurchaseChange } from '../../../hooks/events';
+import { localize } from '../../../../_common/localize';
+import { Explanation } from '../../bottom/explanation';
+import { TimeTooltipWrapper, triggerClick } from '../../../common/helpers';
+
+const AuditSection = ({ data }) => {
+    const auditData = {
+        start: {
+            title  : localize('Contract starts'),
+            content: data?.auditDataStart,
+        },
+        end: {
+            title  : localize('Contract ends'),
+            content: data?.auditDataEnd,
+        },
+        details: {
+            title  : localize('Contract details'),
+            content: data?.auditDataDetails,
+        },
+    };
+
+    return (
+        <>
+            <div className='header-box'>
+                <Button
+                    variant='tertiary'
+                    size='lg'
+                    icon={<LabelPairedArrowLeftMdRegularIcon />}
+                    color='black'
+                    onClick={() => {
+                        purchaseManager.set({
+                            cd_showAudit: false,
+                        });
+                        triggerClick('#contract_purchase_button');
+                    }}
+                />
+                <div className='title-box'>
+                    <Text size='md' bold>
+                        {localize('Audit page')}
+                    </Text>
+                </div>
+                <Button variant='tertiary' size='lg' />
```

Comment:
Probably this button is missing icon and onclick event

Reply:
This dumb button is expected to be there, since the designers centered the title using a button on the left and an empty button on the right. We will be needing to add special styling to position the title if we don't add an empty button on the other side. I can add a comment on this line if that is okay to you?

Reply:
I don't think we should have a dumb button considering accessibility. Could we replace it with a div?

Reply:
did you guys try to replace it with a div? :)